src/Tizen/Avalonia.Tizen/NuiKeyboardHandler.cs

```diff
@@ -52,14 +53,18 @@ private void SendKeyEvent(Window.KeyEventArgs e, Input.Key mapped)
                 _view.InputRoot,
                 type,
                 mapped,
-                modifiers));
+                modifiers,
+                PhysicalKey.None,
+                deviceType,
+                e.Key.KeyString
+                ));
     }
 
     private bool ShouldSendKeyEvent(Window.KeyEventArgs e, out global::Tizen.Uix.InputMethod.KeyCode keyCode)
     {
-        if (string.IsNullOrEmpty(e.Key.KeyString) || 
+        if (string.IsNullOrEmpty(e.Key.KeyString) ||
             e.Key.KeyPressedName is "Delete" or "BackSpace" ||
-            e.Key.IsCtrlModifier() || 
+            e.Key.IsCtrlModifier() ||
             e.Key.IsAltModifier())
         {
             if (Enum.TryParse(e.Key.KeyPressedName, true, out keyCode) ||
```

Comment:
nit: This Enum.TryParse probably can be pre-baked with a HashSet ahead of time, so we won't parse enums on each key input.